ui/packages/catalog-ui-search/src/main/webapp/js/model/Workspace.collection.js

```diff
@@ -79,6 +79,8 @@ module.exports = Backbone.Collection.extend({
       title = '*'
     } else {
       cqlQuery = "anyText ILIKE '" + cql.translateUserqlToCql(text) + "'"
+      // bubbles up failure to parent if CQL cannot be parsed
+      cql.read(cqlQuery)
```

Comment:
Only issue I see here is that if this function is called from somewhere other than `workspaces-templates-container.tsx` we might seem some strange behavior. Not sure if that'll ever happen, but just something to note. The strange behavior would be that if this gets fed an invalid CQL and called from another file that isn't `workspaces-templates-container.tsx`, then we would be experiencing the same issue all over again.